src/py42/services/detectionlists/user_profile.py

```diff
@@ -9,17 +9,15 @@ class DetectionListUserService(BaseService):
     `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
     """
 
-    _api_version = u"v2"
-    _uri_prefix = u"/svc/api/{}".format(_api_version)
-    _resource = u"/user"
+    _resource = u"v2/user"
 
     def __init__(self, connection, user_context, user_service):
         super(DetectionListUserService, self).__init__(connection)
         self._user_context = user_context
         self._user_service = user_service
 
     def _make_uri(self, action):
-        return u"{}{}{}".format(self._uri_prefix, self._resource, action)
+        return u"{}{}".format(self._resource, action)
```

Comment:
It is a tad strange that we have the methods include the slash when calling this helper method. Not a big deal, but just odd.